awscli/customizations/wizard/core.py

```diff
@@ -548,3 +571,31 @@ def _deep_merge(self, original, newvalue):
             return result
         else:
             return newvalue
+
+
+class LoadDataStep(ExecutorStep):
+    NAME = 'load-data'
+
+    def run_step(self, step_definition, parameters):
+        var_resolver = VariableResolver()
+        value = var_resolver.resolve_variables(
+            parameters, step_definition['value'],
+        )
+        format_type = step_definition['load_type']
+        if format_type == 'json':
+            loaded_value = json.loads(value)
+            parameters[step_definition['output_var']] = loaded_value
```

Comment:
Probably it makes sense to raise an exception if format isn't supported instead of just exit